dist/blockstack.js

```diff
@@ -9734,7 +9734,7 @@ module.exports={
   "_args": [
     [
       "bigi@1.4.2",
-      "/Users/larry/git/blockstack.js"
+      "/Users/brunobernardino/Projects/blockstack.js"
```

Comment:
How can I get rid of these? I `git checkout --`d `docs.json` and `package-lock.json` but apparently that still leaked here. 😬

Reply:
Ah. It was this: https://github.com/blockstack/blockstack.js/issues/368